core/controller/src/main/scala/whisk/core/controller/RestAPIs.scala

```diff
@@ -137,8 +163,8 @@ protected[controller] class RestAPIVersion_v1(
             sendCorsHeaders {
                 (pathEndOrSingleSlash & get) {
                     complete(OK, info)
-                } ~ authenticate(basicauth) {
-                    user =>
+                } ~ authenticate(basicauth) { user =>
+                    checkPutSize(config, transid) {
```

Comment:
I would move this even further into the `Entities` handler - I'll send a proposed alternative.